tune_sklearn/tune_basesearch.py

```diff
@@ -67,13 +69,47 @@ def resolve_early_stopping(early_stopping, max_iters):
                         f"or tune scheduler. Got {type(early_stopping)}.")
 
 
+def resolve_loggers(loggers, logdir, logger_config):
+    if loggers is None:
+        return None
+
+    if not isinstance(loggers, list):
+        raise TypeError("`loggers` must be a list of str or tune " "loggers.")
+
+    init_loggers = []
+    logdir = os.path.realpath(logdir)
+    for log in loggers:
+        if isinstance(log, str) and log in TuneBaseSearchCV.defined_loggers:
+            if log == "UnifiedLogger":
+                init_loggers.append(UnifiedLogger(logger_config, logdir))
+            elif log == "TBXLogger":
+                init_loggers.append(TBXLogger(logger_config, logdir))
+            elif log == "JsonLogger":
+                init_loggers.append(JsonLogger(logger_config, logdir))
+            elif log == "CSVLogger":
+                init_loggers.append(CSVLogger(logger_config, logdir))
+            elif log == "MLFLowLogger":
+                init_loggers.append(MLFLowLogger(logger_config, logdir))
+        elif isinstance(log, logger):
+            init_loggers.append(log)
```

Comment:
what about just - "csv", "json", "tensorboard" - and automatically resolve to the logger class?

Reply:
Like take out all other loggers? Aren't these 3 the default loggers though

Reply:
Sorry for the slow reply - basically, in comparison to Tune, let's do the following:

1. default to NO logger (tune.run(loggers=None))
2. User can do: `TuneSearchCV(loggers="csv")` or `TuneSearchCV(loggers=("csv", "tensorboard", "json"))`

if user passes in `"csv"`, you do:

```
init_loggers.append(CSVLogger)
```

Does that make sense?

Reply:
Yeah that makes sense. One thing: I thought json was required to have the analysis object we use to find the best metrics.

Reply:
oh good point; guess it makes sense to default to always having json